src/renderer/components/+helm-charts/helm-chart-details.test.tsx

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+
+import type { AsyncFnMock } from "@async-fn/jest";
+import asyncFn from "@async-fn/jest";
+import type { DiContainer } from "@ogre-tools/injectable";
+import type { RenderResult } from "@testing-library/react";
+import React from "react";
+import directoryForLensLocalStorageInjectable from "../../../common/directory-for-lens-local-storage/directory-for-lens-local-storage.injectable";
+import { HelmChart } from "../../../common/k8s-api/endpoints/helm-charts.api";
+import { getDiForUnitTesting } from "../../getDiForUnitTesting";
+import { noop } from "../../utils";
+import type { CreateInstallChartTab } from "../dock/install-chart/create-install-chart-tab.injectable";
+import createInstallChartTabInjectable from "../dock/install-chart/create-install-chart-tab.injectable";
+import { Notifications } from "../notifications";
+import type { DiRender } from "../test-utils/renderFor";
+import { renderFor } from "../test-utils/renderFor";
+import type { GetChartDetails } from "./get-char-details.injectable";
+import getChartDetailsInjectable from "./get-char-details.injectable";
+import { HelmChartDetails } from "./helm-chart-details";
+
+describe("<HelmChartDetails />", () => {
```

Comment:
As you might have guessed, this should tested as part of behaviour. :) But yeah, I know that this part of application is not that easy to access in behavioural unit tests yet.